src/controller.cpp

```diff
@@ -149,9 +149,6 @@ bool controller::setup_dirs_xdg(const char *env_home, bool silent) {
 
 	config_dir = xdg_config_dir;
 
-	// create data directory if it doesn't exist
-	utils::mkdir_parents(xdg_data_dir, 0700);
```

Comment:
Removing the directory creation both here and in the simple dotfile makes the cache lock fail and newsboat to refuse to start if there are no newsboat directories and newsbeuter migration isn't possible. (Since these directories are not created anywhere else)